ui-admin/src/portal/images/SiteImageList.tsx

```diff
@@ -74,6 +74,17 @@ export default function SiteImageList({ portalContext, portalEnv }:
       className="btn btn-secondary">
       update
     </button>
+  }, {
+    header: '',
+    id: 'copy',
+    cell: ({ row: { original: image } }) => <button onClick={() => {
+      navigator.clipboard.writeText(getImageUrl(portalContext.portal.shortcode,
+        image.cleanFileName,
+        image.version))
+    }}
+    className="btn btn-secondary">
+      <FontAwesomeIcon icon={faClipboard} className="fa-lg"/> Copy Path
```

Comment:
maybe "Copy Url" would be a better name?

Reply:
As-implemented, it's just the path. My thinking was that using the path makes the survey JSON a bit more portable across demo and prod. Since the files are publicly accessible, I wanted to avoid bleeding across deployments.